prow/scripts/cluster-integration/kyma-gke-compass-integration.sh

```diff
@@ -0,0 +1,269 @@
+
+	MSG=""
+	if [[ ${EXIT_STATUS} -ne 0 ]]; then MSG="(exit status: ${EXIT_STATUS})"; fi
+	shout "Job is finished ${MSG}"
+	date
+
+    # Revert previous value for CLUSTER_NAME variable
+    CLUSTER_NAME=${ORIGINAL_CLUSTER_NAME}
+	set -e
+}
+
+function createCluster() {
+	shout "Reserve IP Address for Ingressgateway"
+	date
+	GATEWAY_IP_ADDRESS_NAME="${STANDARIZED_NAME}"
+	GATEWAY_IP_ADDRESS=$(IP_ADDRESS_NAME=${GATEWAY_IP_ADDRESS_NAME} "${TEST_INFRA_CLUSTER_INTEGRATION_SCRIPTS}"/reserve-ip-address.sh)
+	echo "Created IP Address for Ingressgateway: ${GATEWAY_IP_ADDRESS}"
+
+	shout "Create DNS Record for Ingressgateway IP"
+	date
+	GATEWAY_DNS_FULL_NAME="*.${DNS_SUBDOMAIN}.${DNS_DOMAIN}"
+	IP_ADDRESS=${GATEWAY_IP_ADDRESS} DNS_FULL_NAME=${GATEWAY_DNS_FULL_NAME} "${TEST_INFRA_CLUSTER_INTEGRATION_SCRIPTS}"/create-dns-record.sh
+
+	NETWORK_EXISTS=$("${TEST_INFRA_CLUSTER_INTEGRATION_SCRIPTS}/network-exists.sh")
+	if [ "$NETWORK_EXISTS" -gt 0 ]; then
+		shout "Create ${GCLOUD_NETWORK_NAME} network with ${GCLOUD_SUBNET_NAME} subnet"
+		date
+		"${TEST_INFRA_CLUSTER_INTEGRATION_SCRIPTS}/create-network-with-subnet.sh"
+	else
+		shout "Network ${GCLOUD_NETWORK_NAME} exists"
+	fi
+
+	shout "Provision cluster: \"${CLUSTER_NAME}\""
+	date
+
+	if [ -z "$MACHINE_TYPE" ]; then
+		export MACHINE_TYPE="${DEFAULT_MACHINE_TYPE}"
+	fi
+	if [ -z "${CLUSTER_VERSION}" ]; then
+		export CLUSTER_VERSION="${DEFAULT_CLUSTER_VERSION}"
+	fi
+	env ADDITIONAL_LABELS="created-at=${CURRENT_TIMESTAMP}" "${TEST_INFRA_CLUSTER_INTEGRATION_SCRIPTS}"/provision-gke-cluster.sh
+}
+
+function waitUntilInstallerApiAvailable() {
+    shout "Waiting for Installer API"
+
+	attempts=5
+    for ((i=1; i<=attempts; i++)); do
+        numberOfLines=$(kubectl api-versions | grep -c "installer.kyma-project.io")
+        if [[ "$numberOfLines" == "1" ]]; then
+            echo "API found"
+            break
+        elif [[ "${i}" == "${attempts}" ]]; then
+            echo "ERROR: API not found, exit"
+            exit 1
+        fi
+
+        echo "Sleep for 3 seconds"
+        sleep 3
+    done
+}
+
+function installKyma() {
+
+	kymaUnsetVar=false
+
+  # shellcheck disable=SC2043
+	for var in GATEWAY_IP_ADDRESS ; do
+    	if [ -z "${!var}" ] ; then
+        	echo "ERROR: $var is not set"
+        	kymaUnsetVar=true
+    	fi
+	done
+	if [ "${kymaUnsetVar}" = true ] ; then
+    	exit 1
+	fi
+
+	KYMA_INSTALLER_IMAGE="${DOCKER_PUSH_REPOSITORY}${DOCKER_PUSH_DIRECTORY}/${STANDARIZED_NAME}/${REPO_OWNER}/${REPO_NAME}:${CURRENT_TIMESTAMP}"
+
+	shout "Build Kyma-Installer Docker image"
+	date
+	KYMA_INSTALLER_IMAGE="${KYMA_INSTALLER_IMAGE}" "${TEST_INFRA_CLUSTER_INTEGRATION_SCRIPTS}"/create-image.sh
+
+	KYMA_RESOURCES_DIR="${KYMA_SOURCES_DIR}/installation/resources"
+	INSTALLER_YAML="${KYMA_RESOURCES_DIR}/installer.yaml"
+	INSTALLER_CR="/home/prow/go/src/github.com/kyma-project/test-infra/prow/jobs/incubator/compass/resources/installer-cr-cluster-with-compass.yaml.tpl"
```

Comment:
Define this path using ENV variables, for example using `TEST_INFRA_SOURCES_DIR`

Reply:
done